recipes/conda-bash-completion/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "conda-bash-completion" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: "{{ version }}"
```

Comment:
Please do not put the git_rev here. Conda-forge only uses urls. You should simply be able to delete this line.